diff --git a/node_modules/metro-runtime/src/polyfills/require.js b/node_modules/metro-runtime/src/polyfills/require.js
index 2dc0c61..491008d 100644
--- a/node_modules/metro-runtime/src/polyfills/require.js
+++ b/node_modules/metro-runtime/src/polyfills/require.js
@@ -317,14 +317,14 @@ function unknownModuleError(id) {
   }
   return Error(message);
 }
+metroRequire.getModules = () => {
+  return modules;
+};
 if (__DEV__) {
   metroRequire.Systrace = {
     beginEvent: () => {},
     endEvent: () => {},
   };
-  metroRequire.getModules = () => {
-    return modules;
-  };
   var createHotReloadingObject = function () {
     const hot = {
       _acceptCallback: null,
